config/settings/test.yml

```diff
@@ -377,6 +377,15 @@ pdf_generator:
   path: '/form-526ez-pdf-generator/v1/forms/'
   content_type: 'application/vnd.api+json'
 
+evss:
+  dvp:
+    service_name: wss-form526-services-web
+    url: https://csraciapp6.evss.srarad.com
```

Comment:
this seems to legit for me. I would just add some generic fake value here. it just has to match the cassette. and you have it pulling from here, so I'd probably make this look a little less real.